core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1258,6 +1276,15 @@ object KafkaConfig {
       .define(PasswordEncoderCipherAlgorithmProp, STRING, Defaults.PasswordEncoderCipherAlgorithm, LOW, PasswordEncoderCipherAlgorithmDoc)
       .define(PasswordEncoderKeyLengthProp, INT, Defaults.PasswordEncoderKeyLength, atLeast(8), LOW, PasswordEncoderKeyLengthDoc)
       .define(PasswordEncoderIterationsProp, INT, Defaults.PasswordEncoderIterations, atLeast(1024), LOW, PasswordEncoderIterationsDoc)
+
+      /** ********* Raft Quorum Configuration *********/
+      .defineInternal(QuorumVotersProp, STRING, Defaults.QuorumVoters, HIGH)
```

Comment:
Why don't we make the type a LIST like we had previously?

Reply:
With the validation being moved to KafkaConfig, I don't see any reason not to make it a LIST. Will change it back